Public/Sdk/Public/Managed/Shared/frameworks.dsc

```diff
@@ -55,7 +55,7 @@ namespace TargetFrameworks {
     export type DesktopTargetFrameworks = "net451" | "net461" | "net472";
 
     @@public
-    export type CoreClrTargetFrameworks = "netcoreapp2.2";
+    export type CoreClrTargetFrameworks = "netcoreapp3.0";

```

Comment:
Same comment as before: Can we keep netcoreapp2.2? For completeness so that others can keep using it if they need to stay on 2.2?

Reply:
I'll add it back, but this brings back the runtime packages for `netcoreapp2.2` as aliased Nuget references and thus adds more "load" to the initial download process. Also, our self deployment qualifiers are `net472` and `netcoreapp3.0` now - can you elaborate on who 'they' are and what scenario that would entail?

Reply:
note the download only happens when used. Our nuget is still eager but when that one is lazy when not used it should just be a short web request.

---
In reply to: [283756439](https://github.com/microsoft/BuildXL/pull/298#discussion_r283756439) [](ancestors = 283756439)

Reply:
future customers of DScript

---
In reply to: [283913281](https://github.com/microsoft/BuildXL/pull/298#discussion_r283913281) [](ancestors = 283913281,283756439)